coc/entry_logs.py

```diff
@@ -53,7 +53,10 @@ def __getitem__(self, index: int) -> Union[ClanWarLogEntry, RaidLogEntry]:
         items from the endpoint"""
         try:
             ret = self._init_logs[index]
-            return self._model(data=ret, client=self._client, response_retry=self._response_retry)
+            if type(self._model) == type(RaidLogEntry):
```

Comment:
This can be done more elegantly with `if isinstance(self._model, RaidLogEntry):`